js/cdn/sample/src/constants/codeSnippets.ts

```diff
@@ -0,0 +1,253 @@
+          resolve("${APP_CONFIG.authToken}");
+        },
+        onSessionClosed: () => {
+          console.log('Session closed');
+        },
+        onSessionError: (error) => {
+          console.error('Session error:', error);
+        },
+        onSessionRefreshed: () => {
+          console.log('Session refreshed')
+        }
+      }
+    });`
+    : '';
+
+  const contextSection = context
+    ? `
+      context: ${JSON.stringify(context, null, 6).replace(/\n/g, '\n      ')},`
+    : '';
+
+  const languageSection =
+    language !== 'en-US'
+      ? `
+      language: "${language.split('-')[0]}",
+      countryCode: "${language.split('-')[1]}",`
+      : '';
+
+  return `<script src="https://aiagent.sendbird.com/orgs/default/index.js"></script>
+<script>
+  const { loadMessenger } = window;
+  
+  async function initMessenger() {
+    const messenger = await loadMessenger();
+    
+    await messenger.initialize({
+      appId: "${APP_CONFIG.appId}",
+      aiAgentId: "${APP_CONFIG.aiAgentId}"${contextSection}${languageSection}
+    });${userSessionSection}
+    
+    console.log('Messenger ready');
+  }
+  
+  initMessenger();
+</script>`;
+};
+
+export const SETUP_SNIPPETS = {
+  installation: `<!-- Add to your HTML head -->
+<script src="https://aiagent.sendbird.com/orgs/default/index.js"></script>`,
+
+  basicImport: `<!-- Basic setup -->
+<script>
+  const { loadMessenger } = window;
+  // Your messenger code here
+</script>`,
+};
+
+export const CODE_EXAMPLES = {
+  basic: {
+    title: 'Basic Setup',
+    code: `<script src="https://aiagent.sendbird.com/orgs/default/index.js"></script>
+<script>
+  const { loadMessenger } = window;
+  
+  async function initMessenger() {
+    const messenger = await loadMessenger();
+    
+    await messenger.initialize({
+      appId: "${APP_CONFIG.appId}",
+      aiAgentId: "${APP_CONFIG.aiAgentId}"
+    });
+    
+    console.log('Messenger ready');
+  }
+  
+  initMessenger();
+</script>`,
+  },
+
+  authenticated: {
+    title: 'With Authentication',
+    code: `<script src="https://aiagent.sendbird.com/orgs/default/index.js"></script>
+<script>
+  const { loadMessenger } = window;
+  
+  async function initMessenger() {
+    const messenger = await loadMessenger();
+    
+    await messenger.initialize({
+      appId: "${APP_CONFIG.appId}",
+      aiAgentId: "${APP_CONFIG.aiAgentId}"
+    });
+    
+    await messenger.updateUserSession({
+      userId: "${APP_CONFIG.userId}",
+      authToken: "${APP_CONFIG.authToken}",
+      sessionHandler: {
+        onSessionTokenRequired: async (resolve) => {
+          resolve("${APP_CONFIG.authToken}");
+        },
+        onSessionClosed: () => {
+          console.log('Session closed');
+        },
+        onSessionError: (error) => {
+          console.error('Session error:', error);
+        },
+        onSessionRefreshed: () => {
+          console.log('Session refreshed')
+        }
+      }
+    });
```

Comment:
init 쪽에 합쳐서 가이드해야 widget user 안거치고 바로 로그인이 될 것 같심당
```suggestion
      aiAgentId: "${APP_CONFIG.aiAgentId}"
      userSessionInfo: {
        // ...
      }
    });
```